ddtrace/contrib/cyclone/patch.py

```diff
@@ -60,15 +60,20 @@ def traced_requesthandler_on_finish(cyclone, pin, func, instance, args, kwargs):
         return func(*args, **kwargs)
 
     try:
-        span.set_tag("http.method", req.method)
         status_code = instance.get_status()
         if 500 <= int(status_code) < 600:
             span.error = 1
-        span.set_tag("http.status_code", status_code)
-        span.set_tag("http.url", req.full_url().rsplit("?", 1)[0])
+        url = req.full_url().rsplit("?", 1)[0]
         trace_utils.store_response_headers(instance._headers, span, config.cyclone)
-        if config.cyclone.http.trace_query_string:
```

Comment:
Question: is this check for `config.cyclone.http.trace_query_string` required? I noticed that this is not the case for wsgi.

Reply:
Good question! It's not anymore. The check here is also done in `set_http_meta` so it's redundant now 🙂